docs/use.md

```diff
@@ -187,6 +190,119 @@ The [render-as-you-fetch pattern](https://reactjs.org/docs/concurrent-mode-suspe
 
+### Initialize an SDK
+
+A few Firestore SDKs need to be initialized (`firebase().performance().initialize()`, `firebase.remoteConfig().fetchAndActivate()`), or need to have settings set before any other calls are made (`firebase.firestore().enablePersistence()`). This can be done by using an extra argument in the preload method:
+
+```jsx
+preloadFirestore(firestore => {
+  return firestore().enablePersistence();
+});
+```
+
 ### Preload Data
 
 Many ReactFire hooks have corresponding preload functions. For example, you can call `preloadFirestoreDocData` to preload data if a component later calls `useFirestoreDocData`.
+
+### Combining ReactFire Hooks
+
+#### Combine Auth, Firestore, and Cloud Storage to Show a User Profile Card
+
+```jsx
+import {
+  AuthCheck,
+  StorageImage,
+  useFirestoreDocData,
+  useUser,
+  useAuth,
+  useFirestore
+} from 'reactfire';
+
+const DEFAULT_IMAGE_PATH = 'userPhotos/default.jpg';
+
+function ProfileCard() {
+  // get the current user.
+  // this is safe because we've wrapped this component in an `AuthCheck` component.
+  const user = useUser();
+
+  // read the user details from Firestore based on the current user's ID
+  const userDetailsRef = useFirestore()
+    .collection('users')
+    .doc(user.uid);
+  let { commonName, favoriteAnimal, profileImagePath } = useFirestoreDocData(
+    userDetailsRef
+  );
+
+  // defend against null field(s)
+  profileImagePath = profileImagePath || DEFAULT_IMAGE_PATH;
+
+  return (
+    <div>
+      <h1>{commonName}</h1>
+      {/*
+        `StorageImage` converts a Cloud Storage path into a download URL and then renders an image
+       */}
+      <StorageImage style={{ width: '100%' }} storagePath={profileImagePath} />
+      <span>Your favorite animal is the {favoriteAnimal}</span>
+    </div>
+  );
+}
+
+function LogInForm() {
+  const auth = useAuth();
+
+  const signIn = () => {
+    auth().signInWithEmailAndPassword(email, password);
+  };
+
+  return <MySignInForm onSubmit={signIn} />;
+}
+
+function ProfilePage() {
+  return (
+    {/*
+      Render a spinner until components are ready
+    */}
+    <Suspense fallback={<MyLoadingSpinner />}>
+      {/*
+        Render `ProfileCard` only if a user is signed in.
+        Otherwise, render `LoginForm`
+       */}
+      <AuthCheck fallback={<LogInForm />}>{ProfileCard}</AuthCheck>
+    </Suspense>
+  );
+}
+```
+
+### Log Page Views to Google Analytics for Firebase with React Router
+
+```jsx
+import { useAnalytics } from 'reactfire';
+import { Router, Route, Switch } from 'react-router';
+
+function MyPageViewLogger({ location }) {
+  const analytics = useAnalytics();
+
+  // By passing `location.pathname` to the second argument of `useEffect`,
+  // we only log on first render and when the `pathname` changes
+  useEffect(() => {
+    analytics.logEvent('page-view', location.pathname);
```

Comment:
Is this the correct format? Thought it was `'page_view', { ... }`

Reply:
you're right, fixed